readme.md

```diff
@@ -13,7 +15,7 @@ $ npm install --save gifsicle
 ## Usage
 
 ```js
-const {execFile} = require('child_process');
+const execFile = require('child_process').execFile;
```

Comment:
The usage example should target the latest Node.js version, where the previous was allowed. Can you revert this change in all the modules?